src/tools/CMakeLists.txt

```diff
@@ -45,6 +45,31 @@ install(PROGRAMS
   DESTINATION ${CMAKE_INSTALL_LIBDIR}/ceph)
 endif(WITH_TESTS)
 
+set(ceph_conf_srcs ceph_conf.cc)
```

Comment:
i don't want to segment the cmake script by the platforms. could you just avoid building the executables if they are not supported by the MinGW port?

Reply:
We actually need some of the executables (especially rbd-nbd and rbd, others might be nice to have).  True that we may be able to just update the build script and pick the binaries we need so that we don't convolute the cmake file.

Reply:
I've dropped those platform checks, letting the build script pick the binaries that we need. I'm a bit concerned about the tests, in which case picking individual tests might be more cumbersome but we can discuss it later on.